css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These selectors should be in reverse order to demonstrate the ability of specificity to override source order.